### PR TITLE
push notifications

### PR DESCRIPTION
**Pull request by brunomichetti**
Push notification when a match is created.
It hasn't automatic testing.

### CODE REVIEW

File: target/settings.py [hunk 1]
```diff
@@ -188,3 +188,6 @@
 }
 
 SPATIALITE_LIBRARY_PATH = 'mod_spatialite'
+
+ONE_SIGNAL_APP_ID = '7c595075-ffcc-4986-b80c-9ddfaf941d43'
```
Comment (kaozdl) [inline target/settings.py]: Further in development you should handle environment variables with this kind of info.

File: targets/signals.py [hunk 2]
```diff
@@ -9,11 +13,24 @@ def targets_match(t1, t2):
             (t1.radius_in_m + t2.radius_in_m))
 
 
+def send_notification_to_users(user_target, notify_users):
+    onesignal_client = onesignal_sdk.Client()
+    onesignal_client.app = {
+        "app_auth_key": ONE_SIGNAL_AUTH_KEY,
+        "app_id": ONE_SIGNAL_APP_ID
+    }
+    msg = 'Congrats! You have a match with user ' f'{user_target.name}!'
```
Comment (kaozdl) [inline targets/signals.py]: How do you manage message translations?

Comment (brunomichetti) [reply targets/signals.py]: I'm not doing that, I will change it.

File: target/settings.py (hunk 1 above)
Comment (brunomichetti) [reply target/settings.py]: How do I manage this?

Comment (kaozdl) [reply target/settings.py]: you should use os.getenv() to manage enviroment variables

**Pull request merged**